src/Altinn.Apps/KubernetesWrapper/KubernetesWrapper.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="KubernetesClient" Version="8.0.12" />
+    <PackageReference Include="KubernetesClient" Version="8.0.68" />
```

Comment:
Did you manually update all of these packages? Unless you specifically need an updated version of the packages, these should be left for "patching thursday" and updated as part of the regular patching routines.